### PR TITLE
[PROCEDURES] Tweak PR procedures to ease release note construction.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,6 +120,13 @@ Pull requests modifying pre-existing releases should be restricted to bug fixes
 and require at least 2 *+1* binding votes from someone other than the author of
 the pull request with no *-1* binding votes.
 
+In order to facilitate the construction of release notes, the individual merging
+a pull request is responsible for ensuring the pull request either:
+
+- Is tagged on Github with at least one of the tags "bug" or "enhancement"
+- Contains a comment describing why the pull request should not be mentioned in the
+  release notes of the next version of Galaxy
```
Comment (nsoranzo) [inline CONTRIBUTING.md:128]: What about adding a new tag for that instead?

Comment (davebx) [inline CONTRIBUTING.md:128]: I'm in favor of @nsoranzo's suggestion, à la wikipedia's "minor edit" flag.

Comment (jmchilton) [inline CONTRIBUTING.md:128]: I'm not opposed to this at all, I think an earlier draft explicitly mentioned a minor tag opted to keep things simpler. Will make the change and rebase.

**Comment by jmchilton** [comment]
Closing this, but will rebase with new tag language.

**Pull request closed without merging**